tests/sentry/performance_issues/test_sql_injection_detector.py

```diff
@@ -86,3 +86,7 @@ def test_sql_injection_on_laravel_query(self) -> None:
     def test_sql_injection_on_query_with_bindings(self) -> None:
         injection_event = get_event("sql-injection/sql-injection-query-with-bindings")
         assert len(self.find_problems(injection_event)) == 0
+
+    def test_sql_injection_on_event_with_gorm(self) -> None:
```

Comment:
doesn't have to be in this PR, but i wonder if we could update our fixture files and just load them so that way we don't have to write a test per fixture? i'm guessing we have an example of doing this somewhere of our code